core/test/utils/standardize_html_sanitizing_transformers_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:core/presentation/utils/html_transformer/text/standardize_html_sanitizing_transformers.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'dart:convert';
+
+void main() {
+  group('StandardizeHtmlSanitizingTransformers::test', () {
+    const transformer = StandardizeHtmlSanitizingTransformers();
+    const htmlEscape = HtmlEscape();
+
+    test('SHOULD remove attributes of IMG tag WHEN they are invalid', () {
+      const inputHtml = '<img src="1" href="1" onerror="javascript:alert(1)">';
```

Comment:
No enough, onError may call an existing function. Also I think all `on*` properties should be removed

Reply:
Yes, This is just a test case. We have removed the `on*` attributes. We only allow some attributes to be declared in `sanitize_html`

Reply:
Yes, but I was talking about test coverage, you may test more cases

Reply:
Updated unit test for it. Please review again @guimard

Reply:
Better, maybe add same test for any HTML tag (div, span,...), not only IMG

Reply:
Updated. Do you need any other tests? @guimard